lit/docs/operation/opa.lit

```diff
@@ -0,0 +1,233 @@
+\title{Open Policy Agent Integration}{opa}
+
+\use-plugin{concourse-docs}
+
+TODO: blurb about what OPA is and what it can do. Link to OPA
```

Comment:
is this gonna be on a different PR?

Reply:
Refresh! I added it in, its part of this PR.